src/theme/TabItem/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import TabItem from '@theme-original/TabItem';
+import type TabItemType from '@theme/TabItem';
```

Comment:
Do we really need to import both `@theme-original/TabItem` and `@theme/TabItem`? It seems safer to just use `@theme-original/TabItem` if possible, I'm slightly worried about import loops with the `@theme` one.

Reply:
yes, you can - thanks!